plottr/plot/mpl.py

```diff
@@ -1033,12 +1033,11 @@ def _plot1dSinglepanel(self) -> None:
             nAxes = 1
         axes = self._makeAxes(nAxes)
 
-        if len(depvals) > 1:
-            ylbl = self.data.label(depnames[0])
-            phlbl: Optional[str] = f"Arg({depnames[0]})"
-        else:
-            ylbl = None
-            phlbl = None
```

Comment:
It seems like the original intention was to only put in the label if there is only 1 thing on the axis. Is the problem not simply that `if len(depvals) > 1:` should have been `if len(depvals) <= 1:` ?

Reply:
Thanks for the comment.
I considered this one as well, but the only caveat is that with your suggestion, it shows both axis label and figure label if only one plot is selected, which in my opinion is redundant. Nonetheless, your suggested approach also fixes the original problem.

Reply:
Since that was the original intent of the code I think we should keep it

Reply:
Fair enough :). I'll update it then.